kill_windows.go

```diff
@@ -3,20 +3,49 @@
 package kill
 
 import (
+	"golang.org/x/sys/windows"
 	"os"
 	"os/exec"
 	"syscall"
 	"unsafe"
 )
 
+const PROCESS_ALL_ACCESS = windows.STANDARD_RIGHTS_REQUIRED | windows.SYNCHRONIZE | 0xffff
+
+func GetWindowsHandle(pid int) (handle windows.Handle, err error) {
+	handle, err = windows.OpenProcess(PROCESS_ALL_ACCESS, false, uint32(pid))
+	return
+}
+
+func GetCreationTime(pid int) (time int64, err error) {
+	handle, err := GetWindowsHandle(pid)
+	if err != nil {
+		return
+	}
+
```

Comment:
Should there be a `defer CloseHandle(handle)` here?  I think this might leak  otherwise.

Reply:
Thanks for catching this! I added a call to `closeHandle(HANDLE(handle))` to reuse the existing function. I also noticed that `windows.CloseHandle` is available with the new import, though it can also return an error. Not sure if that way would be better or not.

Reply:
I'm still not even a beginner with Go, so I'm not qualified to comment on best practice :)  The only thing that stands out to me is that you never return anything explicitly, while the rest of the code does.  I assume that's purely a stylistic choice and Go does something sensible with the returns (using the parameter variables as they currently are?)

Reply:
> I assume that's purely a stylistic choice and Go does something sensible with the returns (editing previous dumb comment, does it create local variables with the names in the trailing part of the function definition and return those?)

Yes, I was also surprised at how this works but the names before the types in the function return type are local variables which will be used as return values when doing a blank return. Seems to be commonly used to avoid having to supply the full (ok, err) tuple at each return, but I'm certainly not qualified to comment on best practices.

Reply:
for the record I'm not a fan of naked returns (when you specify the variable name as part of the function signature) because they confuse everybody (myself included) who comes across them. I am aware this is my repo so it's strange that the GetProcs function uses naked returns but in truth I hadn't formed this opinion before the PR was originally made (it wasn't made by me)